network/notifee.go

```diff
@@ -91,6 +91,12 @@ func (s *NotifeeService) Connected(_ lp2pnetwork.Network, conn lp2pnetwork.Conn)
 	pid := conn.RemotePeer()
 	s.logger.Info("connected to peer", "pid", pid, "direction", conn.Stat().Direction, "addr", conn.RemoteMultiaddr())
 
+	if conn.Stat().Direction == lp2pnetwork.DirUnknown {
```

Comment:
This is we need to investigate more.

Reply:
I think DirUnknown connections make problems for us. we can't determine whether they are inbound or outbound. In this way, we only accept known connections.

Reply:
In go-libp2p, the `DirUnknown` flag has never been used. Therefore, we are adding code with the assumption that it will never be encountered.

Reply:
SO, if we are 100% sure it's not used. let's close it. I thought we have some unknown Dir connection.